v2/openapi/fb-provider-transfer-api.yaml

```diff
@@ -1283,8 +1384,274 @@ components:
         - succeeded
         - failed
 
-
     DepositAddressCreationPolicy:
       enum:
         - CanCreate
         - CannotCreate
+
+    ParticipantRelationshipType:
+      description: >-
+        Defines the relationship between the Originator and the Beneficiary in a transaction.
+
+        - FirstParty: The Originator and Beneficiary are the same entity.
+        - SecondParty: The Originator and Beneficiary have a direct relationship.
+        - ThirdParty: The transaction involves an intermediary between the Originator and Beneficiary.
+      enum:
+        - FirstParty
+        - SecondParty
+        - ThirdParty
+
+    NationalCountryCode:
+      type: string
+      description: National a2 country codes as defined in ISO 3166.
```

Comment:
As I've already said using "national" in this context makes no sense. You should stick very strictly to the terms used in the ISO standard. There is no "national" and there is no a2. This document is an official communication medium with companies around the world. The terms and the descriptions should be super clear, without any ambiguity; not "almost" precise, but absolutely precise.